### PR TITLE
Add syntax and help to spectate commands

### PR DESCRIPTION
**Pull request by IceNet**
Add syntax and help to spectate commands.

## CHANGES PROPOSED:
-  
-  

## ISSUES ADDRESSED:
None

## TESTS PERFORMED:
Tested ingame and works.

## HOW TO TEST THE CHANGES:
Execute the commands.

## KNOWN ISSUES AND TODO LIST:
None

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Kitzunu** [comment]
I'd rather see the command as "spectate" instead of "spect"

File: data/sql/updates/db_world/2020_04_28_01.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+UPDATE `command` SET `name`='spect leave', `security`='0', `help`='Syntax: .spect leave\r\nLeave spectator mode.' WHERE (`name`='spect leave');
+UPDATE `command` SET `name`='spect reset', `security`='0', `help`='Syntax: .spect reset\r\nSend addon data.' WHERE (`name`='spect reset');
+UPDATE `command` SET `name`='spect spectate', `security`='0', `help`='Syntax: .spect spectate #player\r\nAllow player to spectate arena from another player.' WHERE (`name`='spect spectate');
+UPDATE `command` SET `name`='spect version', `security`='0', `help`='Syntax: .spect version\r\nCheck version of arena spectator addon.' WHERE (`name`='spect version');
+UPDATE `command` SET `name`='spect watch', `security`='0', `help`='Not used.' WHERE (`name`='spect watch');
```
Comment (Kitzunu) [inline data/sql/updates/db_world/2020_04_28_01.sql:5]: Please don't use `''` around numbers

```suggestion
UPDATE `command` SET `name`='spect leave', `security`=0, `help`='Syntax: .spect leave\r\nLeave spectator mode.' WHERE (`name`='spect leave');
UPDATE `command` SET `name`='spect reset', `security`=0, `help`='Syntax: .spect reset\r\nSend addon data.' WHERE (`name`='spect reset');
UPDATE `command` SET `name`='spect spectate', `security`=0, `help`='Syntax: .spect spectate #player\r\nAllow player to spectate arena from another player.' WHERE (`name`='spect spectate');
UPDATE `command` SET `name`='spect version', `security`=0, `help`='Syntax: .spect version\r\nCheck version of arena spectator addon.' WHERE (`name`='spect version');
UPDATE `command` SET `name`='spect watch', `security`=0, `help`='Not used.' WHERE (`name`='spect watch');
```

**Comment by h0lysp4nk** [comment]
Looks good to me, I can confirm that it works in game.

File: data/sql/updates/db_world/2020_04_28_01.sql (hunk 1 above)
Comment (IceNet) [reply data/sql/updates/db_world/2020_04_28_01.sql]: Edited

Seems .spectate doesn't work. Says "There's no such command"
Thats why i used just .spect

**Comment by Kitzunu** [comment]
Change this to spectate instead, it makes no sense to have spect as command

https://github.com/azerothcore/azerothcore-wotlk/blob/4947ba67b0ebd3240ea8f0daecf74b611486f92c/src/server/scripts/Commands/cs_spectator.cpp#L32

**Comment by Kitzunu** [comment]
Also for future PR's, create a new branch with the changes and not on master branch :D

**Comment by IceNet** [comment]
Yes, was my first PR, sry :D

**Comment by Kitzunu** [comment]
Np, thanks for your contributions :D

**Comment by FrancescoBorzi** [comment]
@IceNet thanks for your contribution. However, this is not the right way to push new SQL files, you need to follow this procedure:

http://www.azerothcore.org/wiki/Dealing-with-SQL-files#how-to-create-an-sql-update-file

this way a new **pending SQL** file will be generated and you can include your code inside it. It guarantees that there are no conflicts between SQL update files.

**Comment by IceNet** [comment]
Nevermind. This doesnt work as it should anyway, because before using it, it is necessary to first write in the guild chat /g for the command to be useful. So, for example if i'm spectating someone, and i want to leave arena using command .spec leave i can't.

**Pull request closed without merging**